### ISSUE TITLE
arrow.Arrow.range() includes endpoint

### ISSUE THREAD

**Issue by iynaix** [crsmithdev/arrow#265]
Not sure if this is a bug, but in the example below, the `range()` function includes `end` as well, which is counter to how `range()` in the standard library works, e.g. `range(0, 5)` does not include `5`.

```python
import arrow

start = arrow.get(2014, 8, 4)  # <Arrow [2014-08-04T00:00:00+00:00]>
end = start.replace(days=28)  # <Arrow [2014-09-01T00:00:00+00:00]>
print arrow.Arrow.range("week", start, end)
# [<Arrow [2014-08-04T00:00:00+00:00]>,
# <Arrow [2014-08-11T00:00:00+00:00]>,
# <Arrow [2014-08-18T00:00:00+00:00]>,
# <Arrow [2014-08-25T00:00:00+00:00]>,
# <Arrow [2014-09-01T00:00:00+00:00]>]
```

**Comment by kdeldycke** [comment]
This is fixed by #246 pull request. Just waiting for a merge.

**Comment by andrewelkins** [comment]
Merged
